modules/installation-aws-delete-cluster.adoc

```diff
@@ -0,0 +1,53 @@
+// Module included in the following assemblies:
+//
+// * installing/installing_aws/installing-aws-localzone.adoc
+
+:_content-type: PROCEDURE
+[id="installation-aws-delete-cluster"]
+= Deleting a cluster with a configured AWS Local Zone infrastructure
+
+After you install a cluster on Amazon Web Services (AWS) into an existing Virtual Private Cloud (VPC), and you set subnets for each Local Zone location, you can delete the cluster and any AWS resources associated with it. 
+
+The example in the procedure assumes that you created a VPC and its subnets by using a CloudFormation template.
+
+.Prerequisites
+
+* You can access to the copied CloudFormation template and the stack of AWS resources that you created with the template.
+* You can access the installation configuration file that can deploy your cluster.
+* Optional: If you configured SSH access to your cluster, ensure you know the public and private key pair.
+* You logged in to the AWS CloudFormation CLI console.
+
+.Procedure
+
+. Change to the directory that contains the installation program and delete the cluster by using the `destroy` command:
++
+[source,terminal]
+----
+$ ./openshift-install destroy cluster --dir <installation_directory> \//<1> 
+   --log-level=debug <2>
+----
+<1> For `<installation_directory>`, specify the location of your customized `./install-config.yaml` file.
+<2> To view different log details, specify `error`, `info`, or `warn` instead of `debug`.
+
+. Delete the CloudFormation stack for the Local Zone subnet:
++
+[source,terminal]
+----
+$ aws cloudformation delete-stack --stack-name <local_zone_stack_name>
+----
+
+. Delete the stack of resources that represent the VPC:
++
+[source,terminal]
+----
+$ aws cloudformation delete-stack --stack-name <vpn_stack_name>
+----
+
+.Verification
+
+* Check that you removed the stack resources by issuing the following command in the AWS CLI. The AWS CLI outputs that no template components exist. 
++
+[source,terminal]
+----
+$ aws cloudformation describe-stack --stack-name <local_zone_stack_name> <vpn_stack_name>
```

Comment:
I am not sure if the `--stack-name` receive a list of stacks. The command is also plural: https://docs.aws.amazon.com/cli/latest/reference/cloudformation/describe-stacks.html#options

I think you can add a multi-line here, or just `describe-stacks` without any argument